group/mod/int.go

```diff
@@ -469,3 +468,19 @@ func (i *Int) HideDecode(buf []byte) {
 	i.V.SetBytes(buf)
 	i.V.Mod(&i.V, i.M)
 }
+
+// reverse copies src into dst in byte-reversed order and returns dst,
+// such that src[0] goes into dst[len-1] and vice versa.
+// dst and src may be the same slice but otherwise must not overlap.
+func reverse(dst, src []byte) []byte {
```

Comment:
Now we use it three times - might be worth exporting it here, no?

Reply:
exporting into which package? into util/bytes? The point of kyber is to provide crypto primitives, not byte reversing. A new reader exploring the API should find things in util/ that are related to crypto, like encoding scalars and points, making keys. Not reversing bytes.